integration/integration_test.go

```diff
@@ -213,6 +213,22 @@ func TestExternalCalls(t *testing.T) {
 		return tests
 	})
 }
+func TestPubSub(t *testing.T) {
+	runForRuntimes(t, func(modulesDir string, runtime string, rd runtimeData) []test {
+		return []test{
+			{name: fmt.Sprintf("PubSub%s", rd.testSuffix), assertions: assertions{
+				run("ftl", rd.initOpts...),
+				scaffoldTestData(runtime, "pubsub", rd.modulePath),
```

Comment:
I added these under `pubsub` assuming they can be extended to support `pubsub` once it's ready for integration tests.